tldw_Server_API/app/api/v1/router_groups/minimal.py

```
@@ -0,0 +1,1094 @@
+    ]
+
+
+def _audio_jobs_imports_enabled_for_runtime() -> bool:
+    return not _is_explicit_pytest_runtime() or _env_flag_enabled("MINIMAL_TEST_INCLUDE_AUDIO_JOBS")
+
+
+def _audio_imports_enabled_for_runtime() -> bool:
+    return not _is_explicit_pytest_runtime() or _env_flag_enabled("MINIMAL_TEST_INCLUDE_AUDIO")
+
+
+def iter_minimal_optional_router_specs() -> Iterable[RouterSpec]:
```

Ruff (0.15.11)

[warning] Too many branches (82 > 12)

(PLR0912)